app/src/main/java/com/example/hangouts/homeScreen/MainActivity.java

```diff
@@ -11,4 +15,9 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
     }
+
+    // Temporary logout
+    public void onClickLogout(View view){
+        ParseUser.logOutInBackground();
```

Comment:
What is temp logout? Does this logOutInBackground cause another callback somewhere?